src/sync.rs

```diff
@@ -69,6 +69,14 @@ where
         }
     }
 
+    /// Override the address onto which the firmware is downloaded.
+    ///
+    /// This address is only used if the device uses the DfuSe protocol.
+    pub fn with_override_address(&mut self, address: u32) -> &mut Self {
```

Comment:
The original was called just "override_address". tbh I'm not sure which one is better. Any opinion on this?

Reply:
I put `with_` to be consistent with `with_progress` below, but removing it makes it more consistent with the std library, including the [example](https://rust-lang.github.io/api-guidelines/type-safety.html#non-consuming-builders-preferred) from the guidelines, so I'll change it.